interpreter/core/respond.py

```diff
@@ -112,9 +112,12 @@ def respond(interpreter):
                     interpreter.messages[-1]["language"] = "shell"
 
                 # Get a code interpreter to run it
-                language = interpreter.messages[-1]["language"]
+                language = interpreter.messages[-1]["language"].replace(" ", "")
```

Comment:
I believe the [`language`](https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/llm/setup_openai_coding_llm.py#L127) property in the message object actually comes from the function call the language model makes to the [`execute`](https://github.com/KillianLucas/open-interpreter/blob/main/interpreter/llm/setup_openai_coding_llm.py#L10) function, which only accepts one of the values in this list: `["python", "R", "shell", "applescript", "javascript", "html", "powershell"]`.

If you're looking to adjust how the markdown is generated and remove a potential space between the backticks and the language name that's causing an issue, I believe that would need to be handled in the `message` that comes back from the language model and the `.replace()` call here shouldn't actually have an effect.